### DISCUSSION TITLE
Classified Materials Hosted on GitHub

### DISCUSSION THREAD

**Discussion by Veskaida** [community/community#37008] [Other Feature Feedback, Questions, & Ideas]
There is a GitHub repository which is hosting classified materials, and has been live for a number of days.

I am looking for a way to contact GitHub Staff other than through their GitHub Support system, which I have already submitted a ticket with. The automated system replied about experiencing "high volumes" and "longer than normal wait times", and a significant length of time has passed with the content still accessible.

Are there other means of contacting GitHub staff, to have content removed from the platform? Thanks.

p.s. I am not going to provide any information leading to or about the repository in question except to GitHub staff.

**Comment by radar-blip** [comment]
I'm assuming you've already been here: [Support Page](https://support.github.com/request)

You can also report the user by going to their account and clicking the `block or report` link under their profile picture.

**Reply by Veskaida** [reply]
Yes, on both accounts.

**Reply by radar-blip** [reply]
I dont know of any other way.

**Comment by Patstar210** [comment]
Well under cicumstances I was trying to work on those issues but Im with company and in pubic places I tried to shit my phone off so this wouldn't occur I apologize to the staff and team but it seems tht no matter what I do it creates one issue or another I wanted to work on my core device to eliminate any issue and have a switch to locate and separate my personal use of my device and my work load.. it's an issue we were trying to achieve but my achievement or any plans I was working on all got stripy of my privledge it was a working progress but again for one reason or another someone intercepts and back to square one we go

**Comment by Patstar210** [comment]
Under the circumstances this is all part of a learning experience and how to work as a team I give no consent as owner of my device to remove any software  ingrates on my device prior to being part of your company.. we can all work together which is wht ur team morals are based off opensouce networking which I had contributed to grow your network off my software in my device prior to being part of your team I hope we can all come together as a team for some understanding on both parts

**Comment by LoftyDroid** [comment]
I'm not sure if right but, Can't we simply turn repositories from public to private? so that only the person who has created that repo has the access to it and it wouldn't be visible to others in their profile. 

Let me know if I'm right about it.
